### DISCUSSION TITLE
Upcoming feature poll

### DISCUSSION THREAD

**Discussion by GODrums** [GODrums/BetterFloat#23] [Polls]
## Vote for a feature you want to see in the project scope until BetterFloat V2.

The most voted option is most likely to get implemented first :) 
But of course, I cannot guarantee that all of the features are actually feasible to work on.

If you have completely new ideas, feel free to just post them in the comments and I'll add them to the options.

### Some explanations for available options:
- **7/14/30 day price history / change of an item**: especially for all of you that buy items to resell / "flip" them on Buff later on, this might be an interesting one. This data could be used for example as an indicator of their volatility. Possible for both currently supported marketplaces.
- **Price notifications, if a certain item gets newly listed on a market**, as in it, appears in the newest items tab or with the "live"-feature on Skinport. This would require you to either have the "live"-feature on Skinport or the auto-refresh on CSFloat activated and open in a (background) tab. Such notifications could be for items with a target price, float, or certain sticker. Possible for both CSFloat and Skinport.
- **Skinport: More in-depth data about the attributes of an item**. Skinport itself does not detect blue gems, nr. 1 floats, etc., which are details the extension could add to make the first-glance experience more accurate. CSFloat already does most of this, hence no point in extending this feature past Skinport.
- BetterFloat's ultimate goal is to **support all major marketplaces**. We could continue that with new markets like Bitskins or DMarket. Please comment on your desired website to support next, a comprehensive list can be found here: https://csgoskins.gg/markets

**Comment by Kennyadam** [comment]
can you implement "light mode" csfloat theme like in betterbuff or an inverted version of "default" csfloat theme? i think this will helpful for someone who has eyes that are sensitive to high contrast (like me) or is browsing csfloat in properly lit place and to be able to browse csfloat more comfortably for longer periods of time
